File: Examples/ProgrammersGuide/WorkingWithOCRFeatures/WorkingWithOCRExamples.cs
using Helpers;
using NUnit.Framework;
using System.Reflection;
using Microsoft.VisualStudio.TestTools.UnitTesting;
using Assert = NUnit.Framework.Assert;
using Description = Microsoft.VisualStudio.TestTools.UnitTesting.DescriptionAttribute;

[assembly: AssemblyTitle("Aspose.OCR Examples")]
[assembly: AssemblyDescription("A collection of examples which demonstrate how to use the Aspose.OCR for .NET API.")]
[assembly: AssemblyConfiguration("CSharp")]

namespace Examples.ProgrammersGuide.WorkingWithOCR
{
    [TestClass, TestFixture]
    public class WorkingWithOCRExamples
    {	
        [TestMethod, Test, Owner("Console")]
        public void PerformOCROnImage()
        {
            TestHelper.SetDataDir("ProgrammersGuide/WorkingWithOCR/PerformOCROnImage");

            PerformOCROnImageExample.Program.Main();
        }

        [TestMethod, Test, Owner("Console")]
        public void ExtractingTextPartOfImage()
        {
            TestHelper.SetDataDir("ProgrammersGuide/WorkingWithOCR/ExtractingTextPartOfImage");

            ExtractingTextPartOfImageExample.Program.Main();
        }

    }

    [TestClass, SetUpFixture]
    public class AsposeExamples
    {
        [AssemblyInitialize]
        public static void AssemblyInitialize(Microsoft.VisualStudio.TestTools.UnitTesting.TestContext context)
        {
            Main();
        }

        [SetUp]
        public static void AssemblySetup()
        {
            Main();
        }

        [AssemblyCleanup]
        public static void AssemblyCleanup()
        {
            TestHelper.Cleanup();
        }

        public static void Main()
        {
            // Provides an introduction of how to use this example project.
            TestHelper.ShowIntroForm();
        }
    }
}

File: Examples/ProgrammersGuide/WorkingWithOCR/WorkingWithOCRExamples.vb
Imports System.Reflection
Imports Helpers
Imports NUnit.Framework
Imports Microsoft.VisualStudio.TestTools.UnitTesting
Imports Assert = NUnit.Framework.Assert
Imports Description = Microsoft.VisualStudio.TestTools.UnitTesting.DescriptionAttribute

<Assembly: AssemblyTitle("Aspose.OCR Examples")>
<Assembly: AssemblyDescription("A collection of examples which demonstrate how to use the Aspose.OCR for .NET API.")>
<Assembly: AssemblyConfiguration("VisualBasic")>

Namespace Examples.ProgrammersGuide.WorkingWithOCR
	<TestClass, TestFixture> _
	Public Class WorkingWithOCRExamples
        <TestMethod(), Test(), Owner("Console")> _
	        Public Sub PerformOCROnImage()
	            TestHelper.SetDataDir("ProgrammersGuide/WorkingWithOCR/PerformOCROnImage")
	
	            PerformOCROnImageExample.Program.Main()
        End Sub

        <TestMethod(), Test(), Owner("Console")> _
	        Public Sub ExtractingTextPartOfImage()
	            TestHelper.SetDataDir("ProgrammersGuide/WorkingWithOCR/ExtractingTextPartOfImage")
	
	            ExtractingTextPartOfImageExample.Program.Main()
        End Sub

	End Class

	<TestClass, SetUpFixture> _
	Public Class AsposeExamples
		<AssemblyInitialize> _
		Public Shared Sub AssemblyInitialize(ByVal context As Microsoft.VisualStudio.TestTools.UnitTesting.TestContext)
			Main()
		End Sub

		<SetUp> _
		Public Shared Sub AssemblySetup()
			Main()
		End Sub

		<AssemblyCleanup> _
		Public Shared Sub AssemblyCleanup()
			TestHelper.Cleanup()
		End Sub

		Public Shared Sub Main()
		    ' Provides an introduction of how to use this example project.
			TestHelper.ShowIntroForm()
		End Sub
	End Class
End Namespace